### ISSUE TITLE
MD060 seems to have trouble with multi-byte (e.g. UTF-8) characters

### ISSUE THREAD

**Issue by nsballmann** [igorshubovych/markdownlint-cli#580]
The in v0.46.0 introduced MD060 rule seems to have trouble with multi-byte (e.g. UTF-8) characters.

The following MD table (as far as I can tell correctly aligned):

````markdown
| ID  | ... | Host name / Common Name(CN) | Valid From | Valid To | Content Owner | Server Administrator |         |
| --- | --- | --------------------------- | ---------- | -------- | ------------- | -------------------- | ------- |
| ... | ... | ...                         | ...        | ...      | ...           | ...                  | 🔍 ⌚ ■ |
````

produces the following linting output (with line 120 to be the last line of above table and the 115'th char being the LF after the `|` – I think):

```log
$ markdownlint -d .
.gitlab/issue_templates/Server Certificates.md:120:115 MD060/table-column-style Table column style [Table pipe does not align with heading for style "aligned"]
```

This can be worked around like this:

````markdown
<!-- markdownlint-disable MD060 -->
| ID  | ... | Host name / Common Name(CN) | Valid From | Valid To | Content Owner | Server Administrator |         |
| --- | --- | --------------------------- | ---------- | -------- | ------------- | -------------------- | ------- |
| ... | ... | ...                         | ...        | ...      | ...           | ...                  | 🔍 ⌚ ■ |
<!-- markdownlint-enable MD060 -->
````

I think though, that it would be better (and kind of expected) for `markdownlint` to be able to deal with multi-byte (e.g. UTF-8) characters.
